scripts/zkai_d128_range_policy_discipline_gate.py

```
@@ -0,0 +1,777 @@
+    )
+    return cases
+
+
+def expected_mutation_inventory() -> list[dict[str, str]]:
+    return [
+        {"mutation": mutation, "surface": surface, "rejection_layer": surface}
+        for mutation, surface, _mutated in _mutated_cases(build_core_payload())
+    ]
+
+
+def classify_error(error: Exception) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (7 > 6)

(PLR0911)